api/base.proto

```diff
@@ -35,6 +35,11 @@ message Locality {
   string sub_zone = 3;
 }
 
+// Identifies a percentage, in the range [0.0, 100.0].
```

Comment:
Add more comment to this.

Specifies a percentage, in the range [0.0, 100.0]. For Envoy APIs, this message must be used instead of int32 or double for specifying traffic split and similar settings based on percentage.

PS: if we define a message to replace double, we must force people to use it. Otherwise, we will get mixed double vs Percent, which would be a bad outcome. You may even want to update STYLE.md to force people to use it.

Reply:
Sounds good, I’ll add an update to STYLE.md